ringapp/views.py

```diff
@@ -256,15 +257,38 @@ class PropertyList(ListView):
     model = Property
     
     def get_queryset(self):
-        return Property.objects.order_by('name')
+        props = Property.objects.order_by('name')
+        total = 2*float(Ring.objects.filter(is_commutative=False).count())
+        comm = Ring.objects.filter(is_commutative=True).count()
+        total += comm
+        scores = completeness_scores_ring(commutative_only=False)
+        for obj in props:
+            if obj.id in scores:
+                if obj.commutative_only:
+                    obj.num_known = round(scores[obj.id]/comm, 2)
+                else:
+                    obj.num_known = round(scores[obj.id]/total, 2)
+            else:
+                obj.num_known = 0.0
+        
+        return props
 
 
 class CommPropertyList(ListView):
     model = Property
     template_name = 'ringapp/commproperty_list.html'
 
     def get_queryset(self):
-        return Property.objects.filter(commutative_only=True).order_by('name')
+        props = Property.objects.filter(commutative_only=True).order_by('name')
```

Comment:
At the top of the file,
from django.db.models.functions import Lower
Then on L282, order_by(Lower('name')). 

This gets rid of the case-based sorting issue.

Reply:
@rschwiebert Should I do the same for all 6 lists?

Reply:
And there are also other `order_by`'s in the file, as in L374.